packages/account/src/Sections/Verification/ProofOfIncome/proof-of-income-utils.ts

```diff
@@ -0,0 +1,21 @@
+import { localize } from '@deriv/translations';
+
+export const income_status_codes = {
+    NONE: 'none',
+    PENDING: 'pending',
+    REJECTED: 'rejected',
+    VERIFIED: 'verified',
+    LOCKED: 'locked',
+} as const;
+
+export const poinc_documents_list = [
+    { text: localize('Tax return'), value: 'tax_return' },
+    { text: localize('Employment contract'), value: 'employment_contract' },
+    { text: localize('Payslip'), value: 'payslip' },
+    { text: localize('Country of issue'), value: 'coi' },
+    { text: localize('Business proof of address'), value: 'business_poa' },
+    { text: localize('Article of association'), value: 'article_of_association' },
+    { text: localize('Memorandum'), value: 'memorandum' },
+    { text: localize('Authorisation letter'), value: 'authorisation_letter' },
```

Comment:
is it possible to use <Localize here?

Reply:
not really, Autocomplete component currently expects only strings.